layers/+emacs/org/packages.el

```diff
@@ -1040,6 +1041,21 @@ Headline^^            Visit entry^^               Filter^^                    Da
                   (add-hook 'evil-insert-state-entry-hook #'org-appear-manual-start nil t)
                   (add-hook 'evil-insert-state-exit-hook #'org-appear-manual-stop nil t))))))
 
+(defun org/init-org-transclusion ()
+  (use-package org-transclusion
+    :defer t
+    :init
```

Comment:
Our convention is that, if `:init` or `:config` of `use-package` has multiple forms, they need to be wrapped in a `progn`.